stl/inc/type_traits

```diff
@@ -1440,8 +1441,10 @@ _NODISCARD _MSVC_INTRINSIC constexpr remove_reference_t<_Ty>&& move(_Ty&& _Arg)
 }
 
 _EXPORT_STD template <class _Ty>
-_NODISCARD _MSVC_INTRINSIC constexpr conditional_t<!is_nothrow_move_constructible_v<_Ty> && is_copy_constructible_v<_Ty>, const _Ty&,
-    _Ty&&>
+_NODISCARD
+    _MSVC_INTRINSIC constexpr conditional_t<!is_nothrow_move_constructible_v<_Ty> && is_copy_constructible_v<_Ty>,
+        const _Ty&,
+        _Ty&&>

```

Comment:
Oh this is horrible. I'll hand fix this.

Reply:
I read that change and kind of... 🤮

Reply:
Macros bad, macros confuse clang-format.